bin/addons-linter

```diff
@@ -5,6 +5,13 @@ var path = require('path');
 var absoluteAppRoot = path.resolve(path.join(__dirname, '..'));
 global.appRoot = path.relative(process.cwd(), absoluteAppRoot);
 
-require('../dist/addons-linter')
-  .createInstance({runAsBinary: true})
-  .run();
+try {
+    require('../dist/addons-linter')
```

Comment:
This indentation is off; please make sure you run `npm test` before committing to make sure you don't have any style errors.